### ISSUE TITLE
Firefox: a.remove:focus not behaving as expected

### ISSUE THREAD

**Issue by jobthomas** [woocommerce/storefront#1221]
Brought up here: https://wordpress.org/support/topic/remove-button-outline-goes-to-9999px/#post-12058255

The `:focus` for the remove option in the cart (or the cart widget) is misbehaving on Firefox (can't reproduce on Chrome): the `text-indent: -9999px;` is causing the outline to look odd:

![outline looking weird on firefox](https://cld.wthms.co/6GqEs4+)
Link to image here: https://cld.wthms.co/6GqEs4.

I'm able to reproduce on my test site and the public themes website we have: https://themes.woocommerce.com/storefront/shop/

As a temporary fix I was able to get this working with:

```
a.remove:focus {
    outline: 0px;  
}

a.remove:focus:before {
    outline: 2px solid #96588a;
}
```

In this case I'm basically applying the outline to the before section rather than the whole `a`.

**Comment by jobthomas** [comment]
cc @vedanshujain -- I think my fix should work

**Comment by juliaamosova** [comment]
Can reproduce this in Firefox:

![ff](https://user-images.githubusercontent.com/19143190/68948537-61898200-07b8-11ea-8db5-bd9fbcf61cbd.jpg)

**Comment by StefanNieuwenhuis** [comment]
Why is there a need for a pseudo-element instead of styling the `<a/>` directly?
If the `<a/>` is styled directly, you:
1. improve accessibility
2. make the code more readable and easier to maintain

Proposed styling:
```css
border-radius: 50%;
background: #737781;
line-height: 1.618;

font-weight: 900;
color: white;
text-align: center;
text-decoration: none;
```

**Comment by loic** [comment]
`text-indent: -9999px;` is also causing the focus ring to misbehave on the pagination prev/next buttons on Firefox.

This focus issue issue is so prevalent that I doubt any QA is done on Firefox.

![image (1)](https://user-images.githubusercontent.com/36160/78864212-626fc880-7a65-11ea-8393-d56af1608640.png)

**Comment by Aljullu** [comment]
Thank you all for the feedback. The fix for this issue will be included in Storefront 2.5.6, which is planned to be released in a couple of weeks. If you want, you can already try the Release Candidate:
https://github.com/woocommerce/storefront/releases/tag/version%2F2.5.6-rc.1
We did a lot of internal testing, but if you find an issue, we would appreciate if you could report it.
